website/docs/r/org_ldap.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_ldap"
+sidebar_current: "docs-vcd-resource-org-ldap"
+description: |-
+  Provides a VMware Cloud Director Organization LDAP resource. This can be used to create, delete, and update LDAP configuration for an organization .
+---
+
+# vcd\_org\_ldap
+
+Provides a VMware Cloud Director Org LDAP resource. This can be used to create, update, and delete LDAP configuration for an organization.
+
+Supported in provider *v3.8+*
+
+-> **Note:** This resource requires system administrator privileges.
+
+## Example Usage
+
+```hcl
+provider "vcd" {
+  user     = var.admin_user
+  password = var.admin_password
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+# The settings below (except the server IP) are taken from the LDAP docker testing image
+# https://github.com/rroemhild/docker-test-openldap
+resource "vcd_org_ldap" "my-org-ldap" {
+  org_name  = "my-org"
+  ldap_mode = "CUSTOM"
+  custom_settings {
+    server                  = "192.168.1.172"
+    port                    = 389
+    is_ssl                  = false
+    username                = "cn=admin,dc=planetexpress,dc=com"
+    password                = "GoodNewsEveryone"
+    authentication_method   = "SIMPLE"
+    base_distinguished_name = "dc=planetexpress,dc=com"
+    connector_type          = "OPEN_LDAP"
+    user_attributes {
+      object_class                = "inetOrgPerson"
+      unique_identifier           = "uid"
+      full_name                   = "cn"
+      username                    = "uid"
+      given_name                  = "givenName"
+      surname                     = "sn"
+      telephone                   = "telephoneNumber"
+      group_membership_identifier = "dn"
+      email                       = "mail"
+    }
+    group_attributes {
+      name                        = "cn"
+      object_class                = "group"
+      membership                  = "member"
+      unique_identifier           = "cn"
+      group_membership_identifier = "dn"
+    }
+  }
+  lifecycle {
+    # password value does not get returned by GET
```

Comment:
If we decide to keep the `lifecycle`, we should add a note here, that for updating the password this field needs to be commented out, as there may be many users who are not aware of how this `lifecycle` works.

Reply:
I have changed the example and added a note

Reply:
For some reason I don't see the change

Reply:
updated now